src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/design/mutableexception/Example1.java

```diff
@@ -0,0 +1,47 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="MutableException"/>
+  </module>
+</module>
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.design.mutableexception;
+
+// xdoc section -- start
+class Example1 extends Exception {
```

Comment:
> Changing all classes:

I don't know why you feel like the class names donot work. I can see that there is no more ambiguity here.

Coming to the error part, that is because the violations from the input are not matching the expected ones for parser to bypass.

Reply:
I feel they are not working because there are no errors with original names. But if a single name is changed, the exceptions don't match error shows up. Like there is no other change in the code other than the name. That's why I think it is a problem.